templates/base/Dockerfile.hbs

```
@@ -1,15 +1,13 @@
-FROM node:16
+FROM node:20
```

```suggestion
FROM node:18
```
let's stick to the LTS version